src/components/NavigationSection.svelte

```diff
@@ -69,6 +76,22 @@
     }
   }
 
+  function getCategories (items) {
+    const categoriesOject = {}
+    items.forEach(i => {
+      if (!categoriesOject.hasOwnProperty(i.category)){
+        categoriesOject[i.category] = []
+      }
+      categoriesOject[i.category].push(i)
+    })
+    // since sveltejs doesn't allow to loop through objects
+    const categories = []
+    for (const category in categoriesOject) {
+      categories.push({title: category, items: categoriesOject[category]})
```

Comment:
So that function does not only return a categories array, it also sorts all items into its category… Perhaps you should indicate that in a comment before the function.